drivers/SmartThings/zigbee-illuminance-sensor/capabilities/detectionFrequency.yaml

```diff
@@ -0,0 +1,34 @@
+id: stse.detectionFrequency
```

Comment:
I'm guessing the reason you can't just use a preference here is the same as in other cases: the user has to press a button on the device in order for the interval set command to be received?

Reply:
Yes. This is a similar to:
https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/208#issuecomment-1250581969